spring-cloud-contract-tools/spring-cloud-contract-converters/src/main/java/org/springframework/cloud/contract/verifier/wiremock/DefaultWireMockStubPostProcessor.java

```diff
@@ -43,7 +43,10 @@ public StubMapping postProcess(StubMapping stubMapping, Contract contract) {
 		StubMapping stubMappingFromMetadata = stubMappingFromMetadata(wireMockMetaData.getStubMapping());
 		stubMapping.setResponse(mergedResponse(stubMapping, stubMappingFromMetadata));
 		if (stubMappingFromMetadata.getPostServeActions() != null) {
-			setPostServeActions(stubMapping, stubMappingFromMetadata);
+			//FIXES ISSUE-1727
```

Comment:
You can remove this line cause the commit message will tell us which issue got fixed by this.